runtime/compiler/env/VMJ9.cpp

```diff
@@ -2347,6 +2350,113 @@ TR_J9VMBase::markHotField(TR::Compilation * comp, TR::SymbolReference * symRef,
    }
 
 
+/**
+ * Report a hot field if the JIT has determined that the field has met appropriate thresholds to be determined a hot field. 
+ * Valid if dynamicBreadthFirstScanOrdering is enabled.
+ *
+ * @param reducedCpuUtil normalized cpu utilization of the hot field for the method being compiled
+ * @param clazz pointer to the class where a hot field should be added
+ * @param fieldOffset value of the field offset that should be added as a hot field for the given class
+ * @param reducedFrequency normalized block frequency of the hot field for the method being compiled
+ */
+void
+TR_J9VMBase::reportHotField(int32_t reducedCpuUtil, J9Class* clazz, U_8 fieldOffset,  U_32 reducedFrequency)
+{
+	J9JavaVM * javaVM = _jitConfig->javaVM;
```

Comment:
please format the code with 3 space indent to match the rest of the file since these seem to be tabs.